presto-cli/presto-cli.sh

```diff
@@ -0,0 +1 @@
.201/concurrent-0.201.jar:/Users/xd/.m2/repository/com/facebook/airlift/log/0.201/log-0.201.jar:/Users/xd/.m2/repository/com/facebook/airlift/log-manager/0.201/log-manager-0.201.jar:/Users/xd/.m2/repository/com/facebook/airlift/configuration/0.201/configuration-0.201.jar:/Users/xd/.m2/repository/javax/validation/validation-api/2.0.1.Final/validation-api-2.0.1.Final.jar:/Users/xd/.m2/repository/org/apache/bval/bval-jsr/2.0.0/bval-jsr-2.0.0.jar:/Users/xd/.m2/repository/cglib/cglib-nodep/3.2.5/cglib-nodep-3.2.5.jar:/Users/xd/.m2/repository/org/slf4j/slf4j-jdk14/1.7.25/slf4j-jdk14-1.7.25.jar:/Users/xd/.m2/repository/org/slf4j/slf4j-api/1.7.25/slf4j-api-1.7.25.jar:/Users/xd/.m2/repository/org/slf4j/log4j-over-slf4j/1.7.25/log4j-over-slf4j-1.7.25.jar:/Users/xd/.m2/repository/org/slf4j/jcl-over-slf4j/1.7.25/jcl-over-slf4j-1.7.25.jar:/Users/xd/.m2/repository/ch/qos/logback/logback-core/1.2.3/logback-core-1.2.3.jar:/Users/xd/.m2/repository/com/google/inject/guice/4.2.2/guice-4.2.2.jar:/Users/xd/.m2/repository/aopalliance/aopalliance/1.0/aopalliance-1.0.jar:/Users/xd/.m2/repository/org/weakref/jmxutils/1.19/jmxutils-1.19.jar:/Users/xd/.m2/repository/io/airlift/units/1.3/units-1.3.jar:/Users/xd/.m2/repository/com/google/code/findbugs/jsr305/3.0.2/jsr305-3.0.2.jar:/Users/xd/.m2/repository/javax/inject/javax.inject/1/javax.inject-1.jar:/Users/xd/.m2/repository/com/google/guava/guava/26.0-jre/guava-26.0-jre.jar:/Users/xd/.m2/repository/org/checkerframework/checker-qual/2.5.2/checker-qual-2.5.2.jar:/Users/xd/.m2/repository/com/google/errorprone/error_prone_annotations/2.1.3/error_prone_annotations-2.1.3.jar:/Users/xd/.m2/repository/com/google/j2objc/j2objc-annotations/1.1/j2objc-annotations-1.1.jar:/Users/xd/.m2/repository/org/codehaus/mojo/animal-sniffer-annotations/1.14/animal-sniffer-annotations-1.14.jar:/Users/xd/.m2/repository/jline/jline/2.14.6/jline-2.14.6.jar:/Users/xd/.m2/repository/com/facebook/airlift/json/0.201/json-0.201.jar:/Users/xd/.m2/repository/com/fasterxml/jackson/datatype/jackson-datatype-jdk8/2.10.0/jackson-datatype-jdk8-2.10.0.jar:/Users/xd/.m2/repository/com/fasterxml/jackson/datatype/jackson-datatype-jsr310/2.10.0/jackson-datatype-jsr310-2.10.0.jar:/Users/xd/.m2/repository/com/fasterxml/jackson/datatype/jackson-datatype-guava/2.10.0/jackson-datatype-guava-2.10.0.jar:/Users/xd/.m2/repository/com/fasterxml/jackson/datatype/jackson-datatype-joda/2.10.0/jackson-datatype-joda-2.10.0.jar:/Users/xd/.m2/repository/com/fasterxml/jackson/module/jackson-module-parameter-names/2.10.0/jackson-module-parameter-names-2.10.0.jar:/Users/xd/.m2/repository/com/fasterxml/jackson/dataformat/jackson-dataformat-smile/2.10.0/jackson-dataformat-smile-2.10.0.jar:/Users/xd/.m2/repository/net/sf/opencsv/opencsv/2.3/opencsv-2.3.jar:/Users/xd/.m2/repository/com/squareup/okhttp3/okhttp/3.9.0/okhttp-3.9.0.jar:/Users/xd/.m2/repository/com/squareup/okio/okio/1.13.0/okio-1.13.0.jar:/Applications/IntelliJ IDEA CE.app/Contents/lib/idea_rt.jar com.facebook.presto.cli.Presto
```

Comment:
This one is helping local MacBookPro with ARM M2 chips. Not going to be submitted.